lib/puppetserver/ca/cli.rb

```diff
@@ -28,16 +29,17 @@ class Cli
       INIT_ACTIONS = {
         'import'   => Action::Import,
         'setup'    => Action::Setup,
-        'enable' => Action::Enable,
-        'migrate' => Action::Migrate,
+        'enable'   => Action::Enable,
+        'migrate'  => Action::Migrate
       }
 
       MAINT_ACTIONS = {
         'clean'    => Action::Clean,
         'generate' => Action::Generate,
         'list'     => Action::List,
         'revoke'   => Action::Revoke,
-        'sign'     => Action::Sign
+        'sign'     => Action::Sign,
+        'prune'    => Action::Prune
```

Comment:
hmm. it makes sense semantically to include `prune` in the `MAINT_ACTIONS`, but it then the `ACTION_LIST` below will say that this action requires a running Puppet Server, which is not true. Not sure how we want to address that..

Reply:
we can add in a tidbit like `except for prune`. Or the create a new category but that seems a bit extreme for just one action.

Reply:
yeah, I'm not loving either of those. we can think about it some more and wait to see if anyone else has any good ideas.

Reply:
Almost forgot about this.  Any ideas on what I should do for `prune`?

Reply:
oh yeah, hmm. I think I prefer putting `prune` into `INIT_ACTIONS` over calling out `prune` as an exception in the message for `MAINT_ACTIONS`, but I'm wondering if we should call the `INIT_ACTIONS` something else that would more-obviously include `prune`.. `OFFLINE_ACTIONS`? Not really sure, would love to hear others' opinions on this.

Reply:
I'm absolutely fine with moving it into `INIT_ACTIONS`.